base/jobs/docker/1.0/py3/Dockerfile.cpu

```diff
@@ -111,7 +111,7 @@ RUN ${PIP} install --no-cache --upgrade \
         amazon-braket-ocean-plugin==1.0.9 \
         amazon-braket-schemas==1.10.1 \
         amazon-braket-pennylane-plugin==1.6.7 \
-        amazon-braket-sdk==1.28.0 \
+        amazon-braket-sdk==1.28.1 \
```

Comment:
I checked in NBI it is 1.28.1.dev0, after release is it 1.28.1? Just confirming.

Reply:
Yes, when we built the NBI, the sdk version was 1.28.0 (1.28.1.dev0 if installing from source, as we did). Now the version is 1.28.1 (or 1.28.2.dev0) if you install from source

Reply:
Do we have to update the default simulators as well?